docs/installation.rst

```diff
@@ -43,16 +43,16 @@ If you're upgrading from a previous version, you should review the
 Setting up URLconf
 ------------------
 
-Add the Debug Toolbar's URLs to your project's URLconf as follows::
+Add the Debug Toolbar's URLs to your project's URLconf as the last entry::
```

Comment:
Is there a reason for specifying "the last entry"?

feincms3 uses a catch-all pattern at the end (https://feincms3.readthedocs.io/en/latest/build-your-cms.html#views-and-urls), Wagtail too (https://github.com/wagtail/wagtail/blob/master/wagtail/core/urls.py)

Adding the debug toolbar after those catch-all patterns will not work.
 Anyone using a CMS (feincms3 or wagtail

Reply:
Good point. It doesn't need to strictly be the last entry. I've removed that wording in the latest revision.